src/server/game/Spells/Spell.cpp

```diff
@@ -6632,7 +6632,12 @@ SpellCastResult Spell::CheckItems(uint32* param1 /*= nullptr*/, uint32* param2 /
                     {
                         ItemPosCountVec dest;
                         InventoryResult msg = target->ToPlayer()->CanStoreNewItem(NULL_BAG, NULL_SLOT, dest, m_spellInfo->Effects[i].ItemType, 1);
```

Comment:
should this be called after the check of free space ?

Reply:
yes, **msg** used later for else if

Reply:
my point is, as it's used later, does it need to be called after your new check "if (target->ToPlayer()->GetFreeInventorySpace() == 0)" ? or does it need to be called before for some reason ?

Reply:
I cant put it after new check (before if (msg != EQUIP_ERR_OK)) as first it is needed for "msg" in line 

> if (msg != EQUIP_ERR_OK)